src/components/SearchMovie/SearchMovie.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { MoviesList } from '../MoviesList';
+import moviesFromServer from '../../api/movies.json';
+
+export class SearchMovie extends React.Component {
+  state = {
+    query: '',
+  }
+
+  queryChange = (event) => {
+    this.setState({ query: event.target.value });
+  }
+
+  render() {
+    const { query } = this.state;
+    const visibleMovies = moviesFromServer.filter(
+      movie => movie.title.toLowerCase().includes(query.toLowerCase()),
+    );
```

Comment:
```suggestion
    const { query } = this.state;
    const visibleMovies = moviesFromServer.filter(
      ({ title, description }) => (title.toLowerCase().includes(query) 
      || description.toLowerCase().includes(query)),
    );
```
Ты не досмотрел условие, там указано, что поиск нужно проводить по title и по description